src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -167,7 +170,7 @@ class ExternalDatabaseManagementService(
         return organizationExternalDatabaseColumns.getValue(columnId)
     }
 
-    fun getColumnNamesByTable(orgId: UUID, dbName: String): Map<String, Set<String>> {
+    fun getColumnNamesByTable(dbName: String): Map<String, Set<String>> {
         val columnNamesByTableName = HashMap<String, MutableSet<String>>()
```

Comment:
I think you could create the map with an expected size of `organizationExternalDatabaseTables.size`